venues/MIDL.io/2019/Conference/python/create-metareview-invitations.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python
+
+###############################################################################
+# ex. python create-invitations.py --conf MyConf.org/2017 --baseurl http://localhost:3000
+#       --username admin --password admin_pw
+#
+# To be run after submission due date to create review invitations for all the papers.
+# For each paper:
+# 1) create authorGroup (can see reviews, can't write a review)
+#           reviewer group (reviewers for this paper)
+#           and nonReviewerGroup (folks that aren't allowed to read the review at least not yet)
+# 2) create review invitation
+###############################################################################
+
+## Import statements
+import argparse
+import sys
+import openreview
+from openreview import tools
+import config
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print("Connecting to "+client.baseurl)
+conference = config.get_conference(client)
+
+iterator = tools.iterget_notes(client, invitation=conference.get_submission_id())
+for paper in iterator:
+    paper_num = str(paper.number)
+    paperinv = conference.get_id() + '/-/Paper' + paper_num + '/Meta_Review'
+    paperGroup = conference.get_id() + '/Paper' + paper_num
+
+    ## Confidential Review
+    metareview_reply = {
+        'forum': paper.id,
+        'replyto': paper.id,
+        'writers': {'values-regex': paperGroup + '/Area_Chairs'},
+        'signatures': {'values-regex': paperGroup + '/Area_Chairs'},
```

Comment:
don't you need to set the individual group id as a signature?

like ```Area_Chair1```

Reply:
Take a look to this invitation: https://openreview.net/invitations?id=ICLR.cc/2019/Conference/-/Paper1/Meta_Review